documentation/directory-structure.md

```diff
@@ -211,6 +286,18 @@ This means that each service in the service group must obey the same versioning
 However, multiple separate services can have different versioning cycles, including different SDK packages. Refer to the aforementioned `aks` and `fleet`
 services for examples of different versioning cycles in a service group.
 
+### Migrating from singular service to service group
+
+In case your azure SDK team owned a singular service and is now expanding to a service group, do the following:
+
+- In one pull request, move the singular service to become a service in a service group. This means it will be moved
+  inside a directory denoting the service name (i.e. `/<service>/`), and its README.md file will also be moved there. 
+  As this PR is moving the service, it means the PR diff will include deletions of the singular service README.md and
+  API version folders, and it will include addition of the new README.md and API version folder,
+  both inside the service folder.
```

Comment:
What you describe here is one option and whether to allow this or not is really up to your team and how hard it is for you and your tooling to deal with it. 
Another option is that the 1st service of the 'service group' must stay as it exactly is - this allow diffs to work well.
Then any NEW services go in a subdir with its readme.md & stable/preview folders. 
The end result is that the 1st service in the group is in the parent and ALL OTHER services in the group are in their respective subdirectories.

Your team can pick ONE of these or allow either of these depending on what works best for your team. If you allow moving the 1st service to a subdir, then we would still need all the diff tooling to work correctly and I think this would be hard for your team to accomplish.